packages/example-app-base/src/components/ExampleCard/ExampleCard.tsx

```diff
@@ -80,8 +78,10 @@ export class ExampleCardBase extends React.Component<IExampleCardProps, IExample
             </div>
           );
 
+          const editor = <Editor onChange={this._editorOnChange} width={800} height={500} code={code!} language="typescript" />;
```

Comment:
These values originally came from prototyping, changed width to autofit and kept width static at 500 since autofitting height results in an incredibly small editor

---
In reply to: [302628550](https://github.com/OfficeDev/office-ui-fabric-react/pull/9769#discussion_r302628550) [](ancestors = 302628550)